chapter-10-workload-identities/dbclient/README.md

```diff
@@ -55,7 +55,7 @@ spec:
       - name: svids
         mountPath: /svids
   - name: dbclient-spiffehelper
-    image: dbclient-spiffehelper:v1
+    image: ghcr.io/spiffe/spiffe-helper:0.8.0
```

Comment:
There is now a Docker image so we don't need to build our own image which simplifies the code.